release/kata-deploy-binaries.sh

```diff
@@ -167,6 +167,18 @@ install_firecracker() {
 	popd
 }
 
+# Install static cloud-hypervisor asset
+install_cloud_hypervisor() {
+	info "build static cloud-hypervisor"
+	"${script_dir}/../static-build/cloud-hypervisor/build-static-clh.sh"
+	info "Install static cloud-hypervisor"
+	mkdir -p "${destdir}/opt/kata/bin/"
+	sudo install -D --owner root --group root --mode 0744 cloud-hypervisor/cloud-hypervisor "${destdir}/opt/kata/bin/cloud-hypervisor"
+	pushd "${destdir}"
+	tar -czvf ../kata-cloud-hypervisor-static.tar.gz *
```

Comment:
Our github actions now expects this to be in a particular format "kata-static-{artifact}.tar.gz", so if you go with `install_clh`, the tarball needs to be names as `kata-static-clh.tar.gz` or `kata-static-cloud-hypervisor.tar.gz` for `install_cloud_hypervisor`